api/v1alpha1/lvmvolumegroup_types.go

```diff
@@ -46,6 +46,8 @@ type LVMVolumeGroupSpec struct {
 
 // LVMVolumeGroupStatus defines the observed state of LVMVolumeGroup
 type LVMVolumeGroupStatus struct {
+	// LVMVGStatus contains the status of the VG
+	LVMVGStatus VGStatus `json:"lvmVgStatus,omitempty"`
```

Comment:
Didn't understand why this change is part of the manual disk selection api changes!

Reply:
currently, we do not set any status on the volume group CR. This means we can not communicate the failure to the user easily. So we should set the status and communicate the failures or success via the status

Reply:
We do that in `LVMVolumeGroupNodeStatusSpec` of a different CR.

Reply:
But that is only on a success right not on a failure

Reply:
I removed this as it will create conflict while running vgmanager on the multiple nodes.